Libplanet.Extensions.Cocona/Commands/BlockCommand.cs

```diff
@@ -105,15 +113,38 @@ public enum OutputFormat
         [Argument(Name = "FILE", Description = "File to write the genesis block to.  " +
             "Use `-` for stdout (you can still refer to a file named \"-\" by \"./-\").")]
         string file,
+        [Argument(Name = "VALIDATOR-KEYS", Description = "A public key to use for validating.")]
+        string[] validatorKeys,
```

Comment:
`Option` might be better here, since multiple arguments with different meanings might be confusing.